onnxruntime/test/python/transformers/test_shape_infer_helper.py

```diff
@@ -22,25 +23,22 @@ def _load_onnx(self, model_name):
         import onnx
         return onnx.load_model(model_path)
 
+    #TODO: use a static lightweight model for test
     def test_bert_shape_infer_helper(self):
         model = self._load_onnx("bert-base-cased")
         shape_infer_helper = SymbolicShapeInferenceHelper(model)
         self.assertEqual(shape_infer_helper.infer({"batch_size": 4, "seq_len": 16}), True)
-        self.assertEqual(shape_infer_helper.get_edge_shape("802"), [4, 16, 768])
-        self.assertEqual(shape_infer_helper.get_edge_shape("804"), [4, 16, 1])
-        self.assertEqual(shape_infer_helper.get_edge_shape("1748"), [])
+        self.assertEqual(shape_infer_helper.get_edge_shape("802"), [])
+        self.assertEqual(shape_infer_helper.get_edge_shape("804"), [4, 16, 3072])
+        self.assertEqual(shape_infer_helper.get_edge_shape("1748"), [1])
         self.assertEqual(shape_infer_helper.get_edge_shape("encoder.layer.4.attention.output.LayerNorm.weight"), [768])
-        self.assertEqual(shape_infer_helper.get_edge_shape("1749"), [768, 3072])
-        self.assertEqual(shape_infer_helper.get_edge_shape("817"), [4, 16, 3072])
+        self.assertEqual(shape_infer_helper.get_edge_shape("817"), [4, 16, 1])
         self.assertEqual(shape_infer_helper.get_edge_shape("encoder.layer.4.intermediate.dense.bias"), [3072])
-        self.assertEqual(shape_infer_helper.get_edge_shape("1750"), [3072, 768])
-        self.assertEqual(shape_infer_helper.get_edge_shape("853"), [3])
-        self.assertEqual(shape_infer_helper.get_edge_shape("858"), [1])
-        self.assertEqual(shape_infer_helper.get_edge_shape("880"), [4, 16, 12, 64])
-
-        self.assertEqual(shape_infer_helper.compare_shape("329", "253"), True)
-        self.assertEqual(shape_infer_helper.compare_shape("447", "371"), True)
-        self.assertEqual(shape_infer_helper.compare_shape("329", "817"), False)
+        self.assertEqual(shape_infer_helper.get_edge_shape("880"), [4, 12, 16, 16])
```

Comment:
why the expected shape changed from [4, 16, 12, 64] to [4, 12, 16, 16]? I think the last dimension shall be 64. Is it a bug in recent shape inference change?

Reply:
The graph changed due to transformers update. So the test cases are regenerated